### PR TITLE
Add ec2 softnas alert

### PR DESCRIPTION
**Pull request by tomskelley-gjs**
Title the PR to complete the sentence: "Merging this PR will ..."

Add an alert on the softnas instance if it shows as unhealthy.

### CODE REVIEW

File: infra/terraform/modules/cloudwatch_alerts/cloudwatch.tf [hunk 1]
```diff
@@ -12,7 +12,7 @@ resource "aws_cloudwatch_metric_alarm" "cpu_threshold" {
   statistic           = "Average"
   comparison_operator = "GreaterThanOrEqualToThreshold"
   threshold           = "${var.cpu_threshold}"
-  period              = "${var.period}"
+  evaluation_periods  = "${var.period}"
```
Comment (andylightfoot) [inline infra/terraform/modules/cloudwatch_alerts/cloudwatch.tf:15]: I don't think this should be changed?

File: infra/terraform/modules/cloudwatch_alerts/cloudwatch.tf [hunk 2]
```diff
@@ -47,3 +47,28 @@ resource "aws_cloudwatch_metric_alarm" "cpu_low_threshold" {
...
+  dimensions = {
+    InstanceId = "${element(var.ec2_instance_ids, count.index)}"
+  }
+
+  namespace           = "AWS/EC2"
+  metric_name         = "StatusCheckFailed"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  threshold           = "2"
+  period              = "${var.period}"
+  evaluation_periods  = "${var.evaluation_periods}"
```
Comment (damacus) [inline infra/terraform/modules/cloudwatch_alerts/cloudwatch.tf:66]: As per what @andylightfoot said above, I think this needs to be 

```suggestion
  evaluation_periods  = "${var.period}"
```

File: infra/terraform/modules/cloudwatch_alerts/cloudwatch.tf (hunk 1 above)
Comment (tomskelley-gjs) [reply infra/terraform/modules/cloudwatch_alerts/cloudwatch.tf:15]: Fixed

File: infra/terraform/modules/cloudwatch_alerts/cloudwatch.tf (hunk 2 above)
Comment (tomskelley-gjs) [reply infra/terraform/modules/cloudwatch_alerts/cloudwatch.tf:66]: This one looks OK to me:

  evaluation_periods:                    "3"
      metric_name:                           "StatusCheckFailed"
      namespace:                             "AWS/EC2"
      period:                                "300"

**Review by andylightfoot — APPROVED** [review]

**Pull request merged**